test/unit_Intersection.jl

```diff
@@ -102,6 +102,16 @@ for N in [Float64, Rational{Int}, Float32]
     # absorbing element
     @test absorbing(Intersection) == absorbing(IntersectionArray) == EmptySet
     @test I ∩ E == E ∩ I == IArr ∩ E == E ∩ IArr == E ∩ E == E
+
+    # intersection between a hyperrectangular set and an axis-aligned halfspace
```

Comment:
This file is meant for the lazy `Intersection` but in the end it does not matter.